wordle-web/app.vue

```diff
@@ -1,46 +1,79 @@
 <template>
-  <NuxtLayout>
-    <v-app>
-      <v-app-bar color="primary" :elevation="2">
-        <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
-        </template>
-
-        <v-app-bar-title>WordleApp</v-app-bar-title>
-
-        <v-btn @click="router.push('/')">Home</v-btn>
-        <v-btn @click="router.push('/test')">Test</v-btn>
-        <v-btn icon="mdi-theme-light-dark" @click="toggleTheme" />
-        <v-btn icon="mdi-help-circle" @click="showHelpDialog = true" />
```

Comment:
We added another button here with a trophy mdi icon for the leaderboard page! Could give you a start